docs/sources/view-and-analyze-profile-data/profiling-types/_index.md

```diff
@@ -49,12 +49,12 @@ Various languages support different profiling types. Pyroscope supports the foll
 CPU profiling measures the amount of CPU time consumed by different parts of your application code. High CPU usage can indicate inefficient code, leading to poor performance and increased operational costs. It's used to identify and optimize CPU-intensive functions in your application.
 
 - **When to use**: To identify and optimize CPU-intensive functions
-- **Flamegraph insight**: The width of blocks indicates the CPU time consumed by each function
+- **Flame graph insight**: The width of blocks indicates the CPU time consumed by each function
```

Comment:
Is this a UI element? If so, then the wording should match what is presented in the UI, even if it doesn't match the general usage of "flame graph".

Reply:
Although after reading the other related content to this one, it looks like this isn't a UI element. Never mind about the UI element comment. :)